src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -121,6 +121,32 @@ The features below are only available at the page level. (You can't use them ins
 This is because these features modify the [Response headers](https://developer.mozilla.org/en-US/docs/Glossary/Response_header), which can't be modified after they're sent to the browser. In SSR mode, Astro uses HTML streaming to send each component to the browser as it renders them. This makes sure the user sees your HTML as fast as possible, but it also means that by the time Astro runs your component code, it has already sent the Response headers.
 :::
 
+### `Astro.cookies`
+
+On the `Astro` global, this interface allows to manipulate cookies. It allows to directly get them from requests and set them in responses. Whereas `Astro.request.headers.get('cookie')` gets all cookies and still need to be parsed, `Astro.cookies.get` gets a single cookie in an object.
+
+More details about the API in the reference [Astro.cookies](/en/reference/api-reference/#astrocookies)
+
+The example below shows how to update the value of a counter cookie.
```

Comment:
```suggestion
The example below updates the value of a cookie that updates a page view counter.
```
(I'm assuming that's what this counter is? Is worth mentioning I think because so often we use counters with + and - buttons as examples. That's probably what people think of when they first see "counters" in docs!)  :smile: